server/szurubooru/tests/api/test_post_deleting.py

```diff
@@ -1,12 +1,17 @@
 from unittest.mock import patch
+
 import pytest
+
 from szurubooru import api, db, model, errors
-from szurubooru.func import posts, tags, snapshots
+from szurubooru.func import posts, snapshots
```

Comment:
Unidiomatic imports

Reply:
How would you prefer these imports? It seems like there's a mix of wrapping them in parenthesis and not in the source?

Reply:
I only meant the order, i.e.

```
import system_pkg
from system_pkg import x
import user_pkg
from user_pkg import y
```

W/o the blank lines.

I used `(` only if I had to wrap the line.

Reply:
https://github.com/rr-/szurubooru/pull/156/files#diff-e7bd23942da16f5670a9093e9d6afbe6R1  this view shows it better